app/views/admin/legislations/_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= semantic_form_for [:admin, resource], builder: ActiveAdmin::FormBuilder, html: {'data-controller' => 'check-modified'} do |f| %>
+  <%= f.semantic_errors(*f.object.errors.keys) %>
+
+  <div class="tabs" data-controller="tabs">
+    <ul class="nav nav-tabs" role="tablist">
+      <li><a href="#details">Details</a></li>
+      <li><a href="#documents">Documents</a></li>
+    </ul>
+
+    <%= hidden_field_tag :active_tab, params[:active_tab], 'data-target' => 'tabs.activeTab' %>
+
+    <div class="tab-content">
+      <div id="details">
+        <%= f.inputs do %>
+          <%= f.input :title %>
+          <%= f.input :description, as: :trix %>
+          <%= f.input :document_type_ids, label: 'Document', as: :tags, collection: DocumentType.all %>
+          <%=
+            Arbre::Context.new do
+              columns do
```

Comment:
@tsubik So I've added column layout here without specific requirement, I thought the form will be more compact and easy to scan this way. I guess we can drop columns from here for now until we get more specific details about form UI guidelines.